crates/turborepo-lib/src/commands/daemon.rs

```diff
@@ -60,6 +65,51 @@ pub async fn daemon_client(command: &DaemonCommand, base: &CommandBase) -> Resul
                 println!("Daemon socket file: {}", status.sock_file.to_string_lossy());
             }
         }
+        DaemonCommand::Clean => {
+            // try to connect and shutdown the daemon
+            let client = connector.clone().connect().await;
+            match client {
+                Ok(client) => match client.stop().await {
+                    Ok(_) => {
+                        tracing::trace!("Successfully stopped the daemon");
+                    }
+                    Err(e) => {
+                        tracing::trace!("Unable to stop the daemon: {:?}", e);
+                    }
+                },
+                Err(e) => {
+                    tracing::trace!("Unable to connect to the daemon: {:?}", e);
+                }
+            }
+
+            // remove pid and sock files
+            let mut success = true;
+            trace!("Cleaning up daemon files");
+            // if the pid_file and sock_file still exist, remove them:
+            if pid_file.exists() {
+                let result = std::fs::remove_file(pid_file.clone());
+                // ignore this error
+                if let Err(e) = result {
+                    println!("Failed to remove pid file: {}", e);
+                    println!("Please remove manually: {}", pid_file);
+                    success = false;
+                }
+            }
+            if sock_file.exists() {
+                let result = std::fs::remove_file(sock_file.clone());
+                // ignore this error
+                if let Err(e) = result {
+                    println!("Failed to remove socket file: {}", e);
+                    println!("Please remove manually: {}", sock_file);
+                    success = false;
+                }
+            }
+
+            if success {
+                println!("Success!");
+                println!("Restart the daemon with `turbo daemon start`");
```

Comment:
Do we want to advise the user manually start the daemon instead of just having it get started as part of run?